libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -298,11 +298,13 @@ bool ReferencesResolver::visit(InlineAssembly const& _inlineAssembly)
 			}
 			declarations = m_resolver.nameFromCurrentScope(realName);
 		}
-		if (declarations.size() != 1)
+		if (declarations.size() > 1)
 		{
 			declarationError(_identifier.location, "Multiple matching identifiers. Resolving overloaded identifiers is not supported.");
 			return size_t(-1);
 		}
+		else if (declarations.size() == 0)
+			return size_t(-1);
```

Comment:
Don't know what the policy says, but I'd add a blank line after this

Reply:
The entire function has no blanks. I'd prefer to have some, but perhaps I'll just leave it as is in this file.